[mgmt] recoveryservicesdatareplication release (#34271)
Azure/azure-sdk-for-js: sdk/recoveryservicesdatareplication/arm-recoveryservicesdatareplication/CHANGELOG.md

# Release History
    
## 1.0.0 (2025-05-12)

### Features Added

> The package of @azure/arm-recoveryservicesdatareplication is using our next generation design principles. To learn more, please refer to our documentation [Quick Start](https://aka.ms/azsdk/js/mgmt/quickstart).

Author: ```suggestion
This is the first stable version with the package of @azure/arm-recoveryservicesdatareplication
```
